### PR TITLE
docs: document authentication for system api

### LINKED ISSUE #4005: Document System API usage
We need to document how to use the SystemAPI:
- [ ] "direct" JWT Profile Authentication
  - [ ] link to zitadel-tools for manual tries
 - [ ]  configuration for System Owners

### PR DESCRIPTION
**Pull request by livio-a**
closes #4005

### CODE REVIEW

**Review by fforootd — CHANGES_REQUESTED** [review]

File: docs/docs/guides/integrate/access-zitadel-system-api.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+     <tr>
+        <td>Prerequisites</td>
+        <td>
+            <ul>
+                <li>A <a href="/docs/guides/deploy/overview">self-hosted ZITADEL</a></li>
+            </ul>
+        </td>
+    </tr>
+</table>
+
```
[the remark this answers is not in the document]
Comment (livio-a) [reply docs/docs/guides/integrate/access-zitadel-system-api.md:29]: 👍  i've already removed the question part at the end

**Review by hifabienne — COMMENTED** [review]
A link from the api descriptions of system api to the guide would be great

**Comment by livio-a** [comment]
> A link from the api descriptions of system api to the guide would be great

I've just added one 😄

**Comment by livio-a** [comment]
> A link from the api descriptions of system api to the guide would be great

you mean from the proto description?

File: docs/docs/guides/integrate/access-zitadel-system-api.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  - system-user-1:
+      Path: /system-user-1.pub
+```
+
+or with a base64 encoded value of the key:
+
+```yaml
+SystemAPIUsers:
+  - system-user-1:
+      KeyData: <base64 encoded value of system-user-1.pem>
```
Comment (eliobischof) [inline docs/docs/guides/integrate/access-zitadel-system-api.md:49]:
```suggestion
      KeyData: <base64 encoded value of system-user-1.pub>
```

**Review by eliobischof — DISMISSED** [review]

**Comment by livio-a** [comment]
@fforootd @hifabienne @eliobischof thanks for the reviews... I have accepted / implemented the proposals

**Review by fforootd — APPROVED** [review]

**Review by hifabienne — APPROVED** [review]

**Pull request merged**